app/build.gradle.kts

```diff
@@ -0,0 +1,220 @@
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+    id("com.google.gms.google-services")
+    id("com.google.devtools.ksp") version "1.7.0-1.0.6"
+    kotlin("kapt")
```

Comment:
Didn't know this was a thing, usually we'd put `id("kotlin-kapt")` 😮 
Would this also work for the other `kotlin-x` plugins below? like `kotlin("android")` and `kotlin("parcelize")`? 🤔

Reply:
I also found out from stack overflow 😆